anonymizer/base.py

```diff
@@ -227,13 +227,13 @@ class Anonymizer(object):
 
     faker = DjangoFaker()
 
-    def get_query_set(self):
+    def get_queryset(self):
         """
         Returns the QuerySet to be manipulated
         """
         if self.model is None:
             raise Exception("'model' attribute must be set")
-        qs = self.model._default_manager.get_query_set()
+        qs = self.model._default_manager.get_queryset()
```

Comment:
this is the only line that actually needs to be updated probably but in an effort of consistency, i renamed them all